components/more_direct_channels/more_direct_channels.test.tsx

```diff
@@ -223,35 +220,30 @@ describe('components/MoreDirectChannels', () => {
     });
 
     test('should exclude deleted users if there is not direct channel between users', () => {
-        const users = [
+        const users: UserProfile[] = [
             {
+                ...mockedUser,
                 id: 'user_id_1',
-                label: 'user_id_1',
-                value: 'user_id_1',
                 delete_at: 0,
             },
             {
+                ...mockedUser,
                 id: 'user_id_2',
-                label: 'user_id_2',
-                value: 'user_id_2',
                 delete_at: 0,
             },
             {
+                ...mockedUser,
                 id: 'deleted_user_1',
-                label: 'deleted_user_id_1',
-                value: 'deleted_user_id_1',
                 delete_at: 1,
             },
             {
+                ...mockedUser,
                 id: 'deleted_user_2',
-                label: 'deleted_user_id_2',
-                value: 'deleted_user_id_2',
                 delete_at: 1,
             },
             {
+                ...mockedUser,
                 id: 'deleted_user_3',
-                label: 'deleted_user_id_3',
```

Comment:
@AbhinavVihan Why are we removing these

Reply:
coz UserProfile does not contain these keys and we don't actually need these here.